notebooks/125-torchvision-zoo-to-openvino/125-convnext-classification.ipynb

```diff
@@ -0,0 +1,363 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    from datasets import load_dataset
</pre><p>is it necessary introduce dependency on hf ecosystem for torchvision? as we can use any image as example, maybe better do not load whole dataset, just use some image URL?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1345/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
removed